src/material/datepicker/multi-year-view.spec.ts

```diff
@@ -387,3 +408,16 @@ class MultiYearViewWithMinMaxDate {
   minDate: Date | null;
   maxDate: Date | null;
 }
+
+
+@Component({
+  template: `
+    <mat-multi-year-view [activeDate]="activeDate" [dateClass]="dateClass"></mat-multi-year-view>
+  `
+})
+class MultiYearViewWithDateClass {
+  activeDate = new Date(2017, JAN, 1);
+  dateClass(date: Date) {
```

Comment:
can we have a test that uses the second parameter as well?

Reply:
Good call, adding the test actually caught something that I missed while copying code around.